src/csvcubed/models/validatedmodel.py

```diff
@@ -0,0 +1,42 @@
+# this class will be class attributes and returns an array of errors
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import Any, Callable, Dict, List
+
+from csvcubedmodels.dataclassbase import DataClassBase
+from csvcubed.models.validationerror import ValidateModelProperiesError
+from csvcubed.utils.validations import (
+    validate_str_type,
```

Comment:
Can you only import the functions we definitely need to use? I know the int and list ones aren't necessary here.

Reply:
done